packages/client/wallets/smart-wallet/README.md

```diff
@@ -0,0 +1,261 @@
+# Smart Wallet SDK
+
+A client-side SDK for interacting with Crossmint Smart Wallets. This SDK enables developers to create and manage smart contract wallets that support both traditional keypair-based signing and passkey authentication.
+
+## Installation
+
+```sh
+npm install @crossmint/client-sdk-smart-wallet
+# or
+pnpm add @crossmint/client-sdk-smart-wallet
+```
+
+## Authentication
+
+This SDK uses JSON Web Tokens (JWT) to authenticate your users. JWT is generated by the auth server and used to authorize users to deploy and operate their wallets.
```

Comment:
it cant be any JWT, right? needs to be linked to a crossmint user?

Reply:
right, there is a link to the guide in the next paragraph.

open to reword it!